client/signup/steps/about/index.jsx

```diff
@@ -612,6 +612,7 @@ class AboutStep extends Component {
 			<StepWrapper
 				flowName={ flowName }
 				stepName={ stepName }
+				className={ classNames( 'step-wrapper__two-columns' ) }
```

Comment:
Since displaying content in two columns is now part of `StepWrapper` functionality, I'd suggest we make the two-column mode a prop and hide the class name details inside. That way, the call site will only need to do something like:
```
<StepWrapper
...
twoColumns
/>
```
One more possible way is to expand `stepContent` so that we can assign content components to each column explicitly. 

Whichever way, I think we shouldn't have to assign the same class name on different call sites, which is error-prone.